### PR TITLE
feat(kit): `Stringify` add new pipe

### PR DESCRIPTION
**Pull request by waterplea**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [ ] No

### CODE REVIEW

File: projects/kit/pipes/stringify/stringify.pipe.spec.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {TuiStringifyPipe} from './stringify.pipe';
+
+describe('TuiStringify pipe', () => {
+    it('works', () => {
+        const pipe = new TuiStringifyPipe();
+
+        expect(pipe.transform('data')({data: 237, value: 327})).toEqual('237');
```
Comment (splincode) [inline projects/kit/pipes/stringify/stringify.pipe.spec.ts:7]: What about pass with `null`, `undefined` or `empty string` to transform? ))

File: projects/kit/pipes/stringify/stringify.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import {TuiStringHandler} from '@taiga-ui/cdk';
+
+@Pipe({
+    name: 'tuiStringify',
+})
+export class TuiStringifyPipe implements PipeTransform {
+    public transform<K extends string>(
+        key: K,
+    ): TuiStringHandler<Readonly<Record<K, unknown> & Record<any, any>>> {
+        return value => String(value[key] ?? '');
```
Comment (splincode) [inline projects/kit/pipes/stringify/stringify.pipe.ts:11]: `??` it's work for old version typescript?

Comment (waterplea) [reply projects/kit/pipes/stringify/stringify.pipe.ts:11]: Yes, it works with our TS.

File: projects/kit/pipes/stringify/stringify.pipe.spec.ts (hunk 1 above)
Comment (waterplea) [reply projects/kit/pipes/stringify/stringify.pipe.spec.ts:7]: Added a couple more tests

Comment (splincode) [reply projects/kit/pipes/stringify/stringify.pipe.spec.ts:7]: what about test with invalid key)) not `data`

**Review by nsbarsukov — APPROVED** [review]

File: projects/kit/pipes/stringify/stringify.pipe.spec.ts (hunk 1 above)
Comment (waterplea) [reply projects/kit/pipes/stringify/stringify.pipe.spec.ts:7]: TypeScript would throw

Comment (waterplea) [reply projects/kit/pipes/stringify/stringify.pipe.spec.ts:7]: image

Comment (splincode) [reply projects/kit/pipes/stringify/stringify.pipe.spec.ts:7]: oh, I see)

**Review by vladimirpotekhin — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**